semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badlanguage/error.json

```diff
@@ -3,8 +3,8 @@
     {
       "code": 8,
       "level": "error",
-      "long_msg": "unsupported language rust",
-      "short_msg": "invalid language",
+      "long_msg": "unsupported language: rust. supported languages are: C#, JSON, Json, c, cs, csharp, generic, go, golang, java, javascript, js, json, jsx, lua, ml, none, ocaml, php, py, python, python2, python3, rb, regex, ruby, ts, tsx, typescript",
```

Comment:
Having duplicates (py, python, python2) or (ts, tsx, typescript) probably not ideal

Reply:
I think it's ok actually. Also, some of those are not equivalent (python != python2, tsx != ts) afaik